dashboard-server/api/responses/response.routes.js

```diff
@@ -3,6 +3,7 @@ const controller = require('./response.controller');
 
 router
   .get('/', controller.getAll)
+  .get('/first-and-last', controller.getFirstAndLast)
```

Comment:
Make note that this is a breaking change, so anything in the system that was using the previous get All ("/") will break! In particular the dashboard-client.

Reply:
Good point. I've changed the path back to `/` from `response.routes` but have left the naming on the `controller` as `getFirstAndLast`.